content/docs/engine-dev/tools/tiaf/maintaining-tiaf/debugging-tiaf.md

```diff
@@ -77,47 +87,182 @@ In the log above, you can extract the following information about the AR run and
 
-1. Check out the commit `59e24a62859ce8a932e338f36432e3228621c1ec`.
-2. Use the [storage query tool](./storage-query-tool.md) to retrieve the relevant historic data for this run.
-3. Build the commit.
-4. Place the historic data in the local historic data folder (see the historic workspace entry in any of the tiaf.json files in the TestImpactFramework folder in your build directory).
-5. Run the Python runtime with the same arguments as the TIAF run that failed:
 
-  ```
-  --sequence=tia --fpolicy=continue --suites=main,smoke --labelexcludes=REQUIRES_gpu --changelist=<path to your build folder>\bin\TestImpactFramework\profile\Temp\Python\changelist.76aeff41529f498297bf8194d38d5414.json --report=D:\workspace\o3de\build\windows\bin\TestImpactFramework\profile\Temp\Python\Reports\report.76aeff41529f498297bf8194d38d5414.json --targetout=stdout --testrunner=live
-  ```
+The script configuration file used for this step is as follows:
+
+```json
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: Current File",
+            "type": "python",
+            "request": "launch",
+            "program": "${file}",
+            "console": "integratedTerminal",
+            "justMyCode": true,
+            "args": [
+                "--config", "C:/dev/o3de/build/windows_vs2019/bin/TestImpactFramework/debug/Persistent/tiaf.json", // Replace with your build path
+                "--src-branch", "PR-15830", // src and dst are different as this is a PR build
+                "--dst-branch", "development",
+                "--commit", "85a0bd96daa4bd330d56bd817e8cdaf29d172b63",
+                "--build-number", "001", // The build number can be anything
+                "--suites", "smoke", "main", // The suites must match those used in AR
+                "--label-excludes", "REQUIRES_gpu", // The label exclude filters must match those used in AR
+                "--test-failure-policy", "continue",
+                "--target-output", "stdout",
+                "--runtime-type", "native" // change to "python" for Python TIAF
+               ]
+        }
+    ]
+}
+```
+
```

Comment:
This bullet point underneath the JSON file is also a good place to clarify why we used different source branches across step 1 and step 2
```suggestion

        - It's important to set the `dst-branch` to the same branch used in the previous step but the `src-branch` to the PR build where the AR failed. This is for two reasons: firstly, it ensures that you don't overwrite the historic data (as PR builds are not sources of truth and thus do not store their run data in the persistent storage) and secondly, it is required for generating change lists for unrelated previous and current commits (such as commits from unrelated branches).

    3. Run the TIAF script to generate the change list and arguments used by the TIAF runtime.     
```